src/main/java/com/foalex/bookstore/controller/BookController.java

```diff
@@ -17,35 +20,56 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoints for mapping books")
 @RestController
 @RequiredArgsConstructor
 @RequestMapping("/api/books")
 public class BookController {
     private final BookService bookService;
 
+    @Operation(
+            summary = "Retrieves all books",
+            description = "Retrieves books with pagination and sorting by fields."
+    )
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    public List<BookDto> getAll(Pageable pageable) {
+        return bookService.findAll(pageable);
     }
 
+    @Operation(
+            summary = "Retrieves a book by id",
+            description = "Gets a book with specified id."
+    )
     @GetMapping("/{id}")
     public BookDto getBookById(@PathVariable Long id) {

```

Comment:
you can also validate `@PathVariable` like:
```
@PathVariable @Positive Long id
```

in order to do so - annotate you controller with `@Validated`